doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
I think by "nose-count", @tacaswell meant we don't have a minimum number of reviewers before a PR can be merged and not a reference to nosetests, but it's a bit of an idiom.

Commit bit means membership in the developer role for the repo (generally.)